staging/src/k8s.io/api/resource/v1alpha3/types.go

```diff
@@ -287,6 +295,22 @@ type DeviceAttribute struct {
 	VersionValue *string `json:"version,omitempty" protobuf:"bytes,5,opt,name=version"`
 }
 
+// NullableDeviceAttribute must have exactly one field set.
+// It has the exact same fields as a DeviceAttribute plus `null` as
+// an additional alternative.
+type NullableDeviceAttribute struct {
+	DeviceAttribute `json:",inline" protobuf:"bytes,1,opt,name=deviceAttribute"`
+
+	// NullValue, if set, marks an intentionally empty attribute.
+	//
+	// +optional
+	// +oneOf=ValueType
+	NullValue *NullValue `json:"null,omitempty" protobuf:"bytes,2,opt,name=null"`
```

Comment:
It seems like it might be a problem to have a field named "null" in YAML?  At the very least it's confusing as heck.

Is this really the best we can do?

Reply:
As discussed in a call and [on Slack](https://kubernetes.slack.com/archives/C0409NGC1TK/p1741283312030059), let's put this PR on hold for now. The main focus needs to be on device tainting, for which I have https://github.com/kubernetes/kubernetes/pull/130447 open. It's not ready for review yet, just linking for reference.

/hold
/retitle WIP: Implement DRA admin-controlled attributes (KEP-5027)